client/src/components/Submissions/Submissions.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import { useHistory, useLocation, Link } from 'react-router-dom';
+import { v4 as uuidv4 } from 'uuid';
+import { Button } from '_common';
+import { FileInputDropZone, Section } from '_common';
+import { useSystems } from 'hooks/datafiles';
+import { useUpload } from 'hooks/datafiles/mutations';
+import DataFilesUploadModalListingTable from '../DataFiles/DataFilesModals/DataFilesUploadModalListing/DataFilesUploadModalListingTable.jsx';
+import styles from './Submissions.module.scss';
+
+const SubmissionsUpload = () => {
+  const history = useHistory();
+  const location = useLocation();
+  const reloadCallback = () => {
+    history.push(location.pathname);
+  };
+
+  const { data: allSystems } = useSystems();
+  const submissionSystem = allSystems.find(
+    (s) => s.name === 'APCD Submissions'
```

Comment:
In the interest of generalization to other portals or the possibility that the Submission system name should vary, should we have this look for a substring Submissions?

Reply:
Think it's worth generalizing? My feelings were this was a fairly specific one-off

Reply:
I have no strong feelings, other than that Core Portal is supposed to be "generic" and this is very deployment specific. A different way to have done this would have been to derive the submission system from a system with the 'submission' scheme here: https://github.com/TACC/Core-Portal/blob/d21106592f78d3baa23d2e7bc061d82cacb0e117/server/portal/settings/settings_custom.example.py#L107

Reply:
Yeah we talked about that, but the `scheme` isn't a part of the `_PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEMS` objects, which all rely on the `private` scheme.

Reply:
@Strmiska Two ways we could generalize it:

1. add a new field to items in `_PORTAL_DATA_DEPOT_LOCAL_STORAGE_SYSTEMS` called `type` or `note` or something from which we can specify `submissions`, and use that to render the component
2. go back to your original method of using the substring `submissions` in the system id